packages/server/graphql/intranetSchema/mutations/updateOAuthRefreshTokens.ts

```diff
@@ -0,0 +1,27 @@
+import {GraphQLInt} from 'graphql'
+import {requireSU} from '../../../utils/authorization'
+import {GQLContext} from '../../graphql'
+import getAtlassianAuthsToUpdate from '../../../postgres/queries/getAtlassianAuthsToUpdate'
+import {Threshold} from '~/types/constEnums'
+import {isNotNull} from '../../../utils/predicates'
+
+const updateOAuthRefreshTokens = {
+  type: GraphQLInt,
+  description: `Updates Atlassian OAuth tokens that haven't been updated for more than 14 days`,
+  args: {},
+  resolve: async (_source, {}, {authToken, dataLoader}: GQLContext) => {
+    //AUTH
+    requireSU(authToken)
+
+    // RESOLUTION
+    const olderThan14DaysThreshold = new Date(Date.now() - Threshold.OAUTH_REFRESH)
```

Comment:
let's not put `14Days` in here because OAUTH_REFRESH could change.
maybe just `olderThanThresh`?

Reply:
No longer needed, we have an input param